images/plbase/python-requirements.txt

```diff
@@ -15,33 +15,33 @@ networkx==3.1
 nltk==3.8.1
 numpy==1.25.2
 openpyxl==3.1.2
-pandas-stubs==2.0.2.230605
-pandas==2.0.3
+pandas-stubs==2.0.3.230814
+pandas==2.1.0
```

Comment:
Should stick to pandas 2.0.3 for now if you want to be safe. We want `pandas` to match the first 3 numbers in the `pandas-stubs` version.

Reply:
Done in 092e1d89b.